CONTRIBUTING.md

```diff
@@ -1,3 +1,13 @@
+### Project License:  Eclipse Public License v1.0
+
+- You will only Submit Contributions where You have authored 100% of the content.
+- You will only Submit Contributions to which You have the necessary rights. This means that if You are employed You have received the necessary permissions from Your employer to make the Contributions.
```

Comment:
Well, I'm not sure what this means specifically. I'm by no means an expert of U.S. copyright legislation.

These terms come from Sharon Corbett of the Eclipse Foundation. We could ask her...